docs/web-apps/automated-testing/cypress.md

```diff
@@ -6,184 +6,148 @@ sidebar_label: Cypress
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-This page provides a walkthrough on how to run [Cypress](https://docs.cypress.io/guides/overview/why-cypress.html) tests locally and on Sauce Labs.
+[Cypress](https://docs.cypress.io/guides/overview/why-cypress.html) is a JavaScript front end web application testing framework. `saucectl` -- the Sauce Labs test orchestrator CLI allows you to run Cypress tests directly from your existing Cypress project.
+
+* Don't have Cypress yet? `saucectl` can set up your initial project directory and give you a sample test to run.
+* Already running Cypress? Let `saucectl` run your tests locally in Docker or in the Sauce Labs Cloud, where you have access to thousands of OS/browser combinations and Sauce Labs analytics.  
 
 ## What You'll Need
 
-* A [Sauce Labs](https://saucelabs.com/) account (if you don't have one, start a [free trial](https://saucelabs.com/sign-up))
-* [Download and install `saucectl`](/testrunner-toolkit/installation#installing-testrunner-toolkit)
-    * Ensure you've met the prerequisites outlined in the [Testrunner Toolkit documentation](/testrunner-toolkit/installation#what-youll-need)
-
-## Configure `saucectl`
-
-Before you begin running cypress tests, you must configure `saucectl` if you plan to view test results on the Sauce Labs dashboard or running tests on the Sauce Labs VMs:      
-
-1. Configure your Sauce Labs credentials:
-
-    * via system environment variables:
-        
-        ```bash title="bash example"
-        export SAUCE_USERNAME=<your sauce labs username>
-        export SAUCE_ACCESS_KEY=<your sauce labs access key>
-        ```
-    
-    * via `saucectl configure`:
-    
-        ```bash
-        saucectl configure
-        ```
-      
-      This command generates a `credentials.yml` file:
-      
-      ```yaml title="example credentials.yml"
-      username: <your saucelabs username>
-      accessKey: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-      source: saucectl configure
-      ```
-      
-      :::warning Do NOT commit `credentials.yml`
-      It should go without saying, but do not publicly expose your `credentials.yml` file over the internet. Make sure you add this file to your `.gitignore` file, as you should only use it locally.
-      :::  
-      
-2. Create the `saucectl` configuration:
-    
-    ```bash
-    saucectl new
-    ```
```

Comment:
According to my conversation with Kunal and Alex, this is not the workflow people who are trying to run a test for the first time will use. They will likely just copy-paste their own test into one of the example templates above, and never even use the 'saucectl new' command.